packages/tables/resources/views/components/defer-loading-indicator.php

```diff
@@ -0,0 +1,13 @@
+<div {{ $attributes->class([
+    'filament-tables-empty-state flex flex-1 flex-col items-center justify-center p-6 mx-auto space-y-6 text-center bg-white',
```

Comment:
`filament-tables-empty-state` should not be used here.

Reply:
👍 Should I change it for at filament-tables-defer-loading-indicator class? Or just remove it?

Reply:
Went ahead and added the `filament-tables-defer-loading-indicator` hook.

Reply:
I'm not sure if defer-loading indicator is correct in general, since it's also being used for the search loading indicator, right?

Reply:
It's closer to the empty-state component that includes the loading-indicator component. I was thinking of naming it `defer-loading-state` to match `empty-state`, however the empty state component has 2 different states (loading or no records found), so I didn't feel calling `defer-loading-state` was a good match. But that would be another way to go.

But it's different than just the loading indicator because it has the bg colored circle around it.

Reply:
But inside the table there's no other loading indicator being used, right?

Reply:
And sorry such a small little loading-indicator became such an in-depth conversation when I'm sure there are more important features to be working on 🤦‍♂️

Reply:
Correct, no other loading indicator now